src/DataStructures/Tensor/Expressions/TensorIndexTransformation.hpp

```diff
@@ -151,11 +151,12 @@ transform_multi_index(
       make_array<NumIndicesOut, size_t>(0);
   for (size_t i = 0; i < NumIndicesOut; i++) {
     gsl::at(output_multi_index, i) =
-        (gsl::at(tensorindex_transformation, i) ==
+        // Check that the index is not a time index vs that it is, because it is
```

Comment:
`vs` -> `instead of` and then `because it is` -> `because we expect` (some more fixes to the sentence phrasing to make it real English after my suggestions)

Reply:
Addressed in new fixup https://github.com/sxs-collaboration/spectre/pull/3842/commits/e7d1372c7a18d0a4ca4d791e9507327c0621ef87